src/App.tsx

```diff
@@ -1,24 +1,79 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { TodosFilter } from './components/TodosFiltered/TodosFiltered';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { TodoList } from './components/TodoList/TodoList';
+import { URL, USER_ID } from './utils/Url';
+import { TodoApp } from './components/TodoApp/TodoApp';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    client.get(URL).then(data => {
+      const todosData = data as Todo[];
+
+      setTodos(todosData);
+      setAllTodos(todosData);
+    })
+      .catch(error => {
+        setErrorMessage('Unable to load todos');
+        throw new Error('An error occurred:', error);
```

Comment:
You're throwing an error in the catch block, but there's no handler for this error. It's better to handle this error or remove the throw statement.